configs/nextflow.hopper.config

```diff
@@ -132,13 +138,19 @@ process {
 		publishDir = [ [ path: "/fs1/results/cron/jasen/${params.speciesDir}", mode: 'copy', overwrite: true, pattern: '*.json' ], [ path: "/fs1/results/cron/qc", mode: 'copy', overwrite: true, pattern: '*.cdmpy' ] ]
 	}
 	withName: create_analysis_result {
-		container = "docker://clinicalgenomicslund/bonsai-prp:0.6.0"
+		container = "docker://clinicalgenomicslund/bonsai-prp:0.7.0"
 		publishDir = [ path: "${params.outdir}/${params.speciesDir}/analysis_result", mode: 'copy', overwrite: true ]
 	}
 	withName: create_cdm_input {
-		container = "docker://clinicalgenomicslund/bonsai-prp:0.6.0"
+		container = "docker://clinicalgenomicslund/bonsai-prp:0.7.0"
 		publishDir = [ path: "${params.outdir}/${params.speciesDir}/cdm_input", mode: 'copy', overwrite: true ]
 	}
+	withName: create_yaml {
+		container = "${params.containerDir}/bonsai-prp.sif"
+		publishDir = [ path: "${params.outdir}/${params.speciesDir}/analysis_yaml", mode: 'copy', overwrite: true ]
+		ext.args_prp = {"${params.outdir}/${params.speciesDir}/analysis_result"}
+		ext.args_sourmash = {"${params.outdir}/${params.speciesDir}/sourmash"}
```

Comment:
Not sure about this, but curious: Why are these surrounded with `{}`? From what I find online it creates closures / "anonymous functions" with the internal stuff being set. Is that the intention, and why is that?

Reply:
Ok, well, never mind :)